test/exercism/comment_test.rb

```diff
@@ -26,7 +26,7 @@ def test_mentions
       ["Mention @alice and @bob and also @charlie", %w(alice bob), "ignore unknown users"],
     ].each do |body, mentions, _desc|
       usernames = User.where(id: Comment.new(body: body).mention_ids).map(&:username)
-      assert_equal usernames, mentions
+      assert_equal usernames.sort, mentions.sort
```

Comment:
If you pre-sort the expected values, you only need to explicitly sort the result values.